parquet/src/test/java/org/apache/iceberg/parquet/TestParquetMetrics.java

```diff
@@ -21,14 +21,32 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.ArrayList;
```

Comment:
Looks like imports need to be cleaned up in this PR. This change doesn't use BigDecimal in this file.

Reply:
refactored.